.gitignore

```diff
@@ -0,0 +1 @@
+/.idea
```

Comment:
This should be added to the global or user specific `.gitignore`, not to the project's one.